content/docs/command-reference/gc.md

```diff
@@ -62,9 +62,9 @@ The default remote is cleaned (see `dvc config core.remote`) unless the
   versions of an experiment or project. Note that both options can be combined,
   for example using the `-aT` flag.
 
-- `--all-commits` - same as `-a` or `-T` above, but applies to all Git commits,
-  and in the workspace (implying `-w`). This preserves the cache for all data
-  used in the entire commit history of the project.
+- `--all-commits` - same as `-a` or `-T` above, but applies to all your Git
+  commits, and in the workspace (implying `-w`). This preserves the cache for
+  all data used in the entire commit history of the project.
```

Comment:
- [ ] Should be applied to all DVC commands with `--all-commits` (there's another 5 or 6 left) — separate PR.